cmd/defaults.yaml

```diff
@@ -169,17 +207,25 @@ Projections:
       BulkLimit: 2000
     # The Notifications projection is used for sending emails and SMS to users
     Notifications:
-      # As notification projections don't result in database statements, retries don't have an effect
+      # As notification projections don't result in database statements, retries don't have any effects
       MaxFailureCount: 0
     # The NotificationsQuotas projection is used for calling quota webhooks
     NotificationsQuotas:
       # Delivery guarantee requirements are probably higher for quota webhooks
       # Defaults to 45 days
       HandleActiveInstances: 1080h
-      # As quota notification projections don't result in database statements, retries don't have an effect
+      # As quota notification projections don't result in database statements, retries don't have any effects
       MaxFailureCount: 0
       # Quota notifications are not so time critical. Setting RequeueEvery every five minutes doesn't annoy the db too much.
       RequeueEvery: 300s
+    Analytics:
+      # Analytics delivery guarantee requirements are a bit higher, as they are not interactively retryable
+      # Defaults to 15 days
+      HandleActiveInstances: 360h
```

Comment:
I don't understand this one?

Reply:
@hifabienne this is relevant for analytics events that can't be delivered.

For example, with 15 days by default, if the endpoint returns 4xx or 5xx statuses AND the instance has no events for 15 days, ZITADEL stops retrying.

I'll make that more clear.

Reply:
done